l1-contracts/test/foundry/unit/concrete/Bridgehub/Registry/NewChain.t.sol

```diff
@@ -2,13 +2,13 @@
 
 pragma solidity ^0.8.17;
 
-import {Vm} from "forge-std/Test.sol";
+// import {Vm} from "forge-std/Test.sol";
```

Comment:
There a couple of such instances*

Reply:
I fixed all of them. I commented the import lines because many disabled (commented) tests require those imports.